client/src/pages/dashboard/views/Sandbox.svelte

```diff
@@ -31,17 +31,45 @@
     let showEditCategory = false;
     let showRemoveCategory = false;
     let showActivateCategory = false;
+    let showInsertSnapshot = false;
 
-    const currentlySelected = '';
+    let insertSnapshotAction: number | null = null;
 
     function overflowClickHandler(e: CustomEvent<RowEvent>) {
         if (!e.detail) return;
         currentContext = e.detail;
         showContextMenu = true;
     }
+
+    function contextMenuHandler(e: CustomEvent<RowEvent>) {
+        if (!e.detail) return;
+        switch (e.type) {
+            case Events.SendDocument: {
+                insertSnapshotAction = 0;
+                break;
+            }
+            case Events.TerminateDocument: {
+                insertSnapshotAction = 2;
+                break;
+            }
+            default: break;
+        }
+        if (selectedOffice) showInsertSnapshot = true;
+    }
+
 </script>
 
 <h1>Sandbox</h1>
+{#await userOffices.load()}
+    Loading user offices
+    {:then office}
+        {#if Object.getOwnPropertyNames(office).length !== 0}
+            Select Current Office: <OfficeSelect offices={office} bind:oid={selectedOffice}/>
+        {:else}
+            User does not belong to a single Office.
+        {/if}
```

Comment:
Let's prefer the affirmative for better readability.
```suggestion
        {#if Object.getOwnPropertyNames(office).length === 0}
            User does not belong to a single Office.
        {:else}
            Select Current Office: <OfficeSelect offices={office} bind:oid={selectedOffice}/>
        {/if}
```